deploy/kube-config/riemann/riemann-service.json

```diff
@@ -0,0 +1,36 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "kubernetes.io/cluster-service": "true",
+      "name": "riemann-heapster"
+    },
+    "name": "riemann-heapster"
+  },
+  "spec": {
```

Comment:
Not in the engine itself. There's a separately-distributed [dashboard](https://github.com/aphyr/riemann-dash), which in practice I would package as a separate container. It wouldn't be crazy for it to be in the same pod, though.